### PR TITLE
Issue 2 view entries

### PR DESCRIPTION
**Pull request by sarah-you**
https://user-images.githubusercontent.com/103456134/231642440-b013a0de-6f80-4eeb-b777-729477dcd3aa.mov

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -21,6 +28,78 @@ function saveEntry(event) {
...
+// switch to show or hide no entry text
+const $noEntries = document.querySelector('.center');
+function toggleNoEntries() {
+  if (data.entries.length > 0) {
+    $noEntries.setAttribute('class', 'hidden center');
+  } else {
+    $noEntries.setAttribute('class', 'center');
+  }
 }
+toggleNoEntries();
```
Comment (shawnkost) [inline js/main.js]: You shouldn't be calling it directly like this outside any functions

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Sarah! You just need to redirect to the entries view after you submit the form, and change the one comment below.

**Comment by sarah-you** [comment]
recording with the updates 
[issue-2-view-entries.mov.zip](https://github.com/sarah-you/code-journal/files/11217987/issue-2-view-entries.mov.zip)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**